spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -661,6 +664,41 @@ public int getUnconfirmedCount() {
 		}
 	}
 
+	@Override
+	public void start() {
+	}
+
+	@Override
+	public void stop() {
```

Comment:
`BatchingRabbitTemplate extends RabbitTemplate` but doesn't call `super.stop()` in its `stop()` impl.
Omission? 
Although for safety I'm sure we have to call `super` any way. Or comment somehow why we don't do that...